nano/templates/adv_med.tmpl

```diff
@@ -22,324 +23,298 @@ Used In File(s): \code\game\machinery\adv_med.dm
-	<div class="item">
-		<h4><b>Damage:</b></h4>
-		<div class="itemLabelWide">
-			Brute: 
-		</div>
-		<div class="itemContentMedium">
-			{{:data.occupant.bruteLoss}}
-		</div><br>
-		<div class="itemLabelWide">
-			Burn: 
-		</div>
-		<div class="itemContentMedium">
-			{{:data.occupant.fireLoss}}
-		</div><br>
-		<div class="itemLabelWide">
-			Oxygen: 
-		</div>
-		<div class="itemContentMedium">
-			{{:data.occupant.oxyLoss}}
-		</div><br>
-		<div class="itemLabelWide">
-			Toxins: 
-		</div>
-		<div class="itemContentMedium">
-			{{:data.occupant.toxLoss}}
-		</div><br>
-		<div class="itemLabelWide">
-			Brain:
-		</div>
-		<div class="itemContentMedium">
-			{{:data.occupant.brainLoss}}
-		</div><br>
-		<div class="itemLabelWide">
-			Radiation Level:
-		</div>
-		<div class="itemContentMedium">
-			{{:data.occupant.radLoss}}
-		</div><br>
-		<div class="itemLabelWide">
-			Genetic:
-		</div>
-		<div class="itemContentMedium">
-			{{:data.occupant.cloneLoss}}
-		</div><br>
-		<div class="itemLabelWide">
-			Paralysis:
-		</div>
-		<div class="itemContentMedium">
-			{{:data.occupant.paralysis}}% ({{:data.occupant.paralysisSeconds}} seconds left!)
-		</div><br>
-	</div>
+
 	<div class="item">
-		<div class="itemLabelWide">
-			Body Temperature:
+		<div class="itemLabelNarrow">
+			Temperature:
 		</div>
-		<div class="itemContentMedium">
+		<div class="itemContent">
 			{{:data.occupant.bodyTempC}}&deg;C, {{:data.occupant.bodyTempF}}&deg;F
 		</div>
 	</div>
+
 	{{if data.occupant.hasVirus}}
-		<div class="notice">
+		<span class="bad">
 			Viral pathogen detected in blood stream.
-		</div>
+		</span> <br>
 	{{/if}}
 	{{if data.occupant.hasBorer}}
-		<div class="notice">
-			Large growth detected in frontal lobe, possibly cancerous. Surgical removal is recommended.
-		</div>
+		<span class="bad">
+			Large growth detected in frontal lobe, possibly cancerous. Surgical removal is recommended. </span> <br>
 	{{/if}}
 	{{if data.occupant.blind}}
-		<span class="bad">Pupils unresponsive.</span><br>
+		<span class="average">Pupils unresponsive.</span><br>
 	{{/if}}
 	{{if data.occupant.nearsighted}}
-		<span class="bad">Retinal Misalignment Detected</span>
+		<span class="average">Retinal misalignment detected</span> <br>
 	{{/if}}
 
 	<br>
+	{{:helper.link('Print', 'document', {'print_p' : 1, 'name' : data.occupant.name})}}
+	{{:helper.link('Eject Occupant', 'arrowreturnthick-1-s', {'ejectify' : 1}, data.occupied ? null : 'disabled')}}
+	<br> <br>
 
-	<div style="width: 70%">
-		<div style="width: 50%; text-align:center;"><b>Blood</b></div>
-		<div class="item">
-			<div class="itemLabelWide">
-				Volume
-			</div>
-			<div class="itemContentMedium">
-				{{:data.occupant.blood.volume}}
-			</div>
-		</div>
-		<div class="item">
-			<div class="itemLabelWide">
-				Percent
-			</div>
-			<div class="itemContentMedium">
-				{{:data.occupant.blood.percent}}%
-			</div>
-		</div>
-	</div>
-
+	<b>Damage: </b>
+	<table>
```

Comment:
Could you compress this table section down a bit, please? It takes up a lot of unnecessary space. For example, 
```
<table><tbody>
    <tr>
        <td>Brute Damage:</td>
        <td>Brain Damage:</td>
    </tr>
```

(Also you are missing a `</tbody>`)